### ISSUE TITLE
 I do not get any window listed in the "Capture Window" drop down botton

### ISSUE THREAD

**Issue by zioalex** [H-M-H/Weylus#34]
Hi,
I installed the release 0.8.0 because the 0.9.0 failed. See #33 
I can reach the server config but the drop down button called "Capture window" is empty.
On the server side I think everything is okay:
`
> weylus 
Aug 11 18:57:49.643  INFO weylus::gui: Found more than one IP address for browsers to connect to,
Aug 11 18:57:49.643  INFO weylus::gui: other urls are:
Aug 11 18:57:49.643  INFO weylus::gui: http://192.168.178.185:1701
Aug 11 18:57:49.643  INFO weylus::gui: http://172.19.0.1:1701
Aug 11 18:57:49.643  INFO weylus::gui: http://172.17.0.1:1701
Aug 11 18:57:49.643  INFO weylus::gui: http://192.168.178.179:1701?password=XXXXXX
Aug 11 18:57:49.643  INFO weylus::web: Webserver listening at 0.0.0.0:1701...
Aug 11 19:01:23.942  INFO weylus::web: Client authenticated: 192.168.178.174:44884.
Aug 11 19:01:23.942  INFO weylus::web: Client connected: 192.168.178.174:44884
`
I also do not understand how connection with my laptop will happen.
When the connection is established every cursor movements on the table will be replicated on the laptop?
Thanks

**Comment by H-M-H** [comment]
Until #33 is resolved you can try to just run the binary inside linux.zip on the releases page.
> I also do not understand how connection with my laptop will happen.
When the connection is established every cursor movements on the table will be replicated on the laptop?

Exactly, all touch events will be sent over to your laptop.

**Comment by H-M-H** [comment]
Alright at least some progress :) There are some things you can try to get the capture list working:
- check there is no firewall blocking the websocket port
- try another browser
- try if you can connect to weylus from your computer via localhost
- depending on which browser you use you may be able to (remote) debug things

**Comment by zioalex** [comment]
Yes :-) some progress.

I see the connection on the debug window on my laptop. Therefore I think
that the communication between PC and tablet is ok.
I tried with Chrome and Firefox. Same result.
What is the value of connecting from localhost when I have the connection
established?
Which kind of debug do you have in mind?
It looks like more a permissions issue on the tablet or a glitch in the UI.

**Comment by H-M-H** [comment]
It is possible, that the website loads fine but the communication over websockets does not work as different ports are used. The point of connecting from localhost is to make sure it really is a problem with the tablet and not for example with capturing the windows itself (for example Weylus does not support Wayland or window managers that do not implement EWMH).
On Firefox for Android there is remote debugging: https://developer.mozilla.org/en-US/docs/Tools/about:debugging
And for Chrome on iOS: https://blog.chromium.org/2019/03/debugging-websites-in-chrome-for-ios.html

What kind of tablet do you use, are the browsers up to date?

**Comment by H-M-H** [comment]
Please try the latest build of Weylus and report the output you get from running Weylus like this: `WEYLUS_LOG_LEVEL=TRACE weylus`. Also note the debug log on the browser side. The latest build can be found here: https://github.com/H-M-H/Weylus/actions/runs/232946468

**Comment by zioalex** [comment]
I tried locally, and it works properly. Very nice. I can choose the window to grab without any problem.
Let me try the last build.
My tablet is a Lenovo Yoga Book.

**Comment by zioalex** [comment]
Okay, following the console debug I found out that the port 9001 was not reachable.
In fact I opened only the port 1701. Now I opened also the port 9001 and it works great! :-D thanks.
Would be great to document the needed port.

However I tried with some applications. Gimp works but with Xornal++ I cannot write. It is like I always have the right click and the page continue to pan. The resolution is also not be best but however this would be another issue.
